billing-report-agent/src/main/java/com/epam/pipeline/billingreportagent/service/impl/converter/StorageToBillingRequestConverter.java

```diff
@@ -138,7 +133,11 @@ public AwsStorageToBillingRequestConverter(final AbstractEntityMapper<StorageBil
     }
 
     private Optional<Long> extractStorageSize(final SearchResponse response) {
-        final ParsedSum sumAggResult = response.getAggregations().get(STORAGE_SIZE_AGG_NAME);
+        final Aggregations aggregations = response.getAggregations();
+        if (aggregations == null) {
+            return Optional.empty();
+        }
+        final ParsedSum sumAggResult = aggregations.get(STORAGE_SIZE_AGG_NAME);
         final long storageSize = new Double(sumAggResult.getValue()).longValue();
         final long totalMatches = response.getHits().getTotalHits();
```

Comment:
Let's move `totalMatches` check to the start of the method

Reply:
Done